src/Pipelines/multisample.php

```diff
@@ -252,7 +263,8 @@ function extract_info($format, $data)
 	if (!$annotation_only)
 	{		
 		//main variant calling for autosomes and gonosomes
-		if ($dragen_gvcfs_exist) //gVCFs exist > merge them
+		$use_deepvar = get_path("use_deepvariant");
+		if ($dragen_gvcfs_exist && !$use_deepvar) //gVCFs exist > merge them
```

Comment:
do not check $use_deepvar here. That changes the behaviour!
DeepVariant is just used whereever freebayes would have been used.